.clj-kondo/hooks/clojure/core.clj

```diff
@@ -179,6 +179,134 @@
     (non-thread-safe-form-should-end-with-exclamation* node))
   {:node node})
 
+;; ----------------- Modules -----------------
+
+;; TODO -- this hard coding is not ideal, it would be better to support metadata on the namespaces themselves.
+(def ^:private global-modules
```

Comment:
Besides the convenience of leaving this detail for another PR, it's quite nice seeing all the modules in one place for now, particularly if we want to plug some of the leaks in `metabase.db`.